yarn-project/simulator/src/avm/avm_simulator.test.ts

```diff
@@ -122,37 +122,36 @@ describe('AVM simulator: transpiled Noir contracts', () => {
   });
 
   describe.each([
-    ['sha256_hash', sha256],
-    ['keccak_hash', keccak],
-  ])('Hashes with 2 fields returned in noir contracts', (name: string, hashFunction: (data: Buffer) => Buffer) => {
-    it(`Should execute contract function that performs ${name} hash`, async () => {
-      const calldata = [...Array(10)].map(_ => Fr.random());
-      const hash = hashFunction(Buffer.concat(calldata.map(f => f.toBuffer())));
+    [
+      'sha256_hash',
+      /*input=*/ randomFields(10),
+      /*output=*/ (fields: Field[]) => {
+        const resBuffer = sha256(Buffer.concat(fields.map(f => f.toBuffer())));
+        return [new Fr(resBuffer.subarray(0, 16)), new Fr(resBuffer.subarray(16, 32))];
```

Comment:
nonblocking: this is pretty noisy, maybe we define them just above?

Reply:
will cleanup in a follow up if I can avoid running CI again!